### PR TITLE
sca default implemented

### PR DESCRIPTION
**Pull request by derins**
### Why:
* Handles the no file upload case of Smart Charging Algorithm

### What:
* Accesses the pre generated models on S3 if a user doesn't provide data.
* current values can be found in the below screenshots, let me know if the naming should be changed on the rate structures

Screen Shot 2020-10-06 at 4 03 39 PM
Screen Shot 2020-10-06 at 4 03 44 PM

### CODE REVIEW

**Review by jongoncalves — APPROVED** [review]

File: frontend/src/Components/AlgInputs/AlgInputsLoadControl.js [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+import React, {Component} from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import { withStyles } from '@material-ui/core/styles';
+import { countyRes } from '../Api/CountyData';
+import { loadControlPromise } from '../Api/AlgorithmData';
+import { loadControlDefaultParams } from '../Api/algorithmDefaultParams';
+import { serverUrl } from '../Api/server';
+import axios from 'axios';
```
Comment (avpeery) [inline frontend/src/Components/AlgInputs/AlgInputsLoadControl.js]: Is this formatting from eslint?

**Review by avpeery — APPROVED** [review]
hey nice idea with documenting the functionality with pictures. looks good

File: frontend/src/Components/AlgInputs/AlgInputsLoadControl.js (hunk 1 above)
Comment (derins) [reply frontend/src/Components/AlgInputs/AlgInputsLoadControl.js]: no it was me, does eslint prefer "?

Comment (avpeery) [reply frontend/src/Components/AlgInputs/AlgInputsLoadControl.js]: Yes, I've been using ESLint on script -- just don't want to put in extra work for ourselves formatting in the future, no big deal here. If you have more preferred linter for javascript, let me know! eslint has been seeming to work well for far.

Comment (derins) [reply frontend/src/Components/AlgInputs/AlgInputsLoadControl.js]: I’ll make the change before merging

**Pull request merged**